contents/handbook/growth/sales/crm.md

```diff
@@ -168,15 +164,52 @@ The process is fairly straightforward. A couple of points to note:
+
+Whilst after the demo stage the is largely automated, we can move customers back to the hands-on pipeline if we feel their value criteria has changed.
+
+### PQL Hands-on pipeline
+
+We use Pocus as a Product-led Sales (PLS) tool to give us oversight of new Product Qualified Leads (PQL).  Our criteria for identifying new PQLs is currently:
+
+1. Customers who have not yet set up a billing subscription who are
+   * Predicted to send over 7m events
+   * Predicted to send over 4m events and 200k recordings
+2. Customers who have set up a billing subscription but who haven't paid yet and are
+   * Predicted to have a first bill over $1,666
+   * Predicted to send over 7 million events
+
+Pocus will also provide indicators to how engaged the users in the organization are with the product, so we can identify Power users and Champions.  Our main aim here is to make contact and ensure they are happy with PostHog, ideally setting up a Slack Connect channel and having an intro call with their team.
+
+We also have a deal pipeline for any high-value PQLs:
+
+1. Product Qualified Lead (20%) - We have identified a PQL using Pocus
+2. Contact Made (30%) - We've reached out to them and had a response, and have a meeting booked or Slack connect channel with them.
+3. Meeting Completed (50%) - We've had a meeting with them and understand their evaluation criteria / preferred buying process.
+4. Product Evaluation (80%) - We are actively supporting them to evaluate and be successful with PostHog.  We may have enabled a trial period so that they don't feel burdened by usage limits.
+5. Commercial & Legal Review (90%) - They have completed their evaluation and PostHog is the right tool for them.  We are now working with them on contractual items such as custom pricing, MSAs etc.
+6. Closed won (Won) - They have signed the contract or paid 2 consecutive monthly bills and are officially a PostHog customer.
+7. Closed lost (Lost) - At some point in the pipeline they decided not to use us.  We should make a note as to the reasons why and optionally set a reminder task to follow up with them if we have improvements that could change their mind on our roadmap.
+8. Closed admin (Lost) - Deal was created in error (e.g. a duplicate) but we still want to keep a record of it.
+
 
 ## All done - now what?
 
 This is just the beginning of what will hopefully be an awesome relationship with a new customer!
 
 We are just getting started here, but a few things that you should do:
 - Make sure you invite the customer to our PostHog Users Slack!
-- If they are a large/target customer, they should also already have a Slack Connect channel in our company workspace.
-- Check in with them regularly and ensure they aren't blocked by support/other issues
+- Or if they are a large/target customer, they should also already have a Slack Connect channel in our company workspace.
+- Check in with them regularly and ensure they aren't blocked by support/other issues
```

Comment:
We could be more specific here. Are we aiming for a once-every-two-months catch up? If they aren't interested in that, do we send them a slack message/email every two months?